doc/build-unix.md

```diff
@@ -41,7 +41,7 @@ Optional dependencies:
  Library     | Purpose          | Description
  ------------|------------------|----------------------
  miniupnpc   | UPnP Support     | Firewall-jumping support
- libdb4.8    | Berkeley DB      | Wallet storage (only needed when wallet enabled)
+ libdb4.8    | Berkeley DB      | Optional, wallet storage (only needed when wallet enabled)
```

Comment:
the sentence makes it sound like it is optional because the wallet can be disabled. wouldn't it be better to say "Optional. Needed to open legacy wallets."

Reply:
Probably shouldn't use the word "legacy" (at least user-facing) while it isn't actually legacy yet.